providers/flagd/README.md

```diff
@@ -45,52 +49,32 @@ FlagdProvider flagdProvider = new FlagdProvider(
-| certPath              | FLAGD_SERVER_CERT_PATH         | String                 | null      | Remote & In-process |
-| cache                 | FLAGD_CACHE                    | String - lru, disabled | lru       | Remote              |
-| maxCacheSize          | FLAGD_MAX_CACHE_SIZE           | int                    | 1000      | Remote              |
-| maxEventStreamRetries | FLAGD_MAX_EVENT_STREAM_RETRIES | int                    | 5         | Remote              |
-| retryBackoffMs        | FLAGD_RETRY_BACKOFF_MS         | int                    | 1000      | Remote              |
-| deadline              | FLAGD_DEADLINE_MS              | int                    | 500       | Remote              |
-
-### OpenTelemetry support
-
-OpenTelemetry support can be enabled either through [automatic instrumentation](https://opentelemetry.io/docs/instrumentation/java/automatic/) 
-or with [manual instrumentation](https://opentelemetry.io/docs/instrumentation/java/manual/). 
-
-For manual instrumentation, flagd provider can be constructed with an `OpenTelemetry` instance.
-
-```java
-FlagdOptions options = 
-        FlagdOptions.builder()
-                    .openTelemetry(openTelemetry)
-                    .build();
-
-FlagdProvider flagdProvider = new FlagdProvider(options);
-```
-
-Please refer [OpenTelemetry example](https://opentelemetry.io/docs/instrumentation/java/manual/#example) for best 
-practice guideline.
-
-Telemetry configuration combined with [flagd telemetry ](https://github.com/open-feature/flagd/blob/main/docs/configuration/flagd_telemetry.md)
-allows distributed tracing.
+| --------------------- | ------------------------------ | ---------------------- | --------- | ------------------- |
+| host                  | FLAGD_HOST                     | String                 | localhost | remote & in-process |
+| port                  | FLAGD_PORT                     | int                    | 8013      | remote & in-process |
+| tls                   | FLAGD_TLS                      | boolean                | false     | remote & in-process |
+| socketPath            | FLAGD_SOCKET_PATH              | String                 | null      | remote & in-process |
+| certPath              | FLAGD_SERVER_CERT_PATH         | String                 | null      | remote & in-process |
+| deadline              | FLAGD_DEADLINE_MS              | int                    | 500       | remote & in-process |
+| cache                 | FLAGD_CACHE                    | String - lru, disabled | lru       | remote              |
+| maxCacheSize          | FLAGD_MAX_CACHE_SIZE           | int                    | 1000      | remote              |
+| maxEventStreamRetries | FLAGD_MAX_EVENT_STREAM_RETRIES | int                    | 5         | remote              |
+| retryBackoffMs        | FLAGD_RETRY_BACKOFF_MS         | int                    | 1000      | remote              |
+
+> [!NOTE]  
+> Some configurations are only applicable for RPC resolver.
```

Comment:
The table refers to this as `remote`. We should use the same terminology throughout the doc.

Reply:
Ya I'll update to RPC.